packages/cmd/__tests__/help/WebHelpManager.test.ts

```diff
@@ -10,15 +10,19 @@
 */
 
 import * as fs from "fs";
+import * as fsExtra from "fs-extra";
 import * as path from "path";
 import * as rimraf from "rimraf";
 
+jest.mock("opener");
```

Comment:
Not sure why this Jest mock works when I run the tests on my machine, but not on Jenkins?